docs/integrations/api.md

```diff
@@ -0,0 +1,553 @@
+- `archiveRate`: true/false - Use an exchange rate close to the transaction time when calculating currency values instead of the current one. **true** by default.
+- `onlyRewards`: true/false - Include only rewards without transfers or stakings. **false** by default.
+- `sort`: asc/desc - Sort of transactions by blockTime. **desc** by default.
+- `skip`: number - Amount of data items to skip.
+- `limit`: number - Maximum amount of data items to respond with.
+
+`skip` and `limit` params are used for pagination eg:
+
+```
+skip: 0, limit: 100 = 1 page
+skip: 100, limit: 100 = 2 page
+skip: 200, limit: 100 = 3 page
+```
+
+### Responses
+
+Response body:
+
+```json
+{
+  "events": [],
+  "totals": {
+    "ethRewards": "12345",
+    "currencyRewards": "1.2345"
+  },
+  "averageApr": "4.9123",
+  "ethToStEthRatio": 1.0041070589388954,
+  "stETHCurrencyPrice": {
+    "eth": 0.99590974,
+    "usd": 1876.31
+  },
+  "totalItems": 500
+}
+```
+
+Event types:
+
+- Staking
+- Transfer
+- Reward
+
+Staking event example:
+
+```json
+{
+  "sender": "0x123",
+  "amount": "12345",
+  "shares": "12345",
+  "sharesBefore": "0",
+  "sharesAfter": "12345",
+  "totalPooledEtherBefore": "12345",
+  "totalPooledEtherAfter": "12345",
+  "totalSharesBefore": "12345",
+  "totalSharesAfter": "12345",
+  "balanceAfter": "12345",
+  "block": "12345",
+  "blockTime": "12345",
+  "transactionHash": "0x12345",
+  "transactionIndex": "1",
+  "logIndex": "1",
+  "transactionLogIndex": "1",
+  "type": "staking",
+  "balance": "12345",
+  "change": "12345",
+  "currencyChange": "20.12345",
+  "epochDays": "12345.12345",
+  "epochFullDays": "12345"
+}
+```
+
+Transfer event example:
+
+```json
+{
+  "from": "0x12345",
+  "to": "0x12345",
+  "value": "12345",
+  "shares": "12345",
+  "sharesBeforeDecrease": "12345",
+  "sharesAfterDecrease": "12345",
+  "sharesBeforeIncrease": "12345",
+  "sharesAfterIncrease": "12345",
+  "totalPooledEther": "12345",
+  "totalShares": "12345",
+  "balanceAfterDecrease": "1",
+  "balanceAfterIncrease": "1",
+  "mintWithoutSubmission": false,
+  "block": "12345",
+  "blockTime": "12345",
+  "transactionHash": "0x12345",
+  "transactionIndex": "1",
+  "logIndex": "1",
+  "transactionLogIndex": "1",
+  "type": "transfer",
+  "direction": "in",
+  "balance": "12345",
+  "change": "12345",
+  "currencyChange": "0.0000000000000012345",
+  "epochDays": "12345.12345",
+  "epochFullDays": "12345"
+}
+```
+
+Reward event example:
+
+```json
+{
+  "id": "0x12345",
+  "totalPooledEtherBefore": "12345",
+  "totalPooledEtherAfter": "12345",
+  "totalSharesBefore": "12345",
+  "totalSharesAfter": "12345",
+  "block": "12345",
+  "blockTime": "12345",
+  "logIndex": "1",
+  "type": "reward",
+  "reportShares": "12345",
+  "balance": "12345",
+  "rewards": "12345",
+  "change": "12345",
+  "currencyChange": "0.12345",
+  "epochDays": "12345.12345",
+  "epochFullDays": "12345",
+  "apr": "4.12345"
+}
+```
+
+### Status Codes
```

Comment:
I guess that would be hard to maintain. I suggest referring to swagger or getting some opeanapi autogenerator to md files